shared/control/parallel_dd.control

```diff
@@ -12,13 +12,8 @@ the files system.
 """
 
 # YOU NEED TO SPECIFY A FILESYSTEM
-path = glob.glob("/dev/[hsv]d?")
-path.sort()
-fs = path[-1]
-platform = Popen(["uname", "-m"], stdout=PIPE).communicate()[0]
-if "ppc" in platform:
-    if not Popen("lsscsi", stdout=PIPE).communicate()[0]:
-        fs = path[0]
+path = glob.glob("/dev/disk/by-id/*TARGET_DISK*")
```

Comment:
hi, the following is the part of function called:
workspace/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/generic/tests/cfg/autotest_control.cfg
	- parallel_dd:
		blk_extra_params_stg = "serial=TARGET_DISK0"
		test_timeout = 36000
		images += " stg"
		image_format_stg = qcow2
		boot_drive_stg = yes
		image_name_stg = images/storage
		image_size_stg = 10G
		force_create_image_stg = yes
		test_control_file = parallel_dd.control

workspace/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/generic/tests/autotest_control.py
def run(test, params, env):
	...
	control_path = os.path.join(test.virtdir, "control",params.get("test_control_file"))  ==> test_control_file = parallel_dd.control
	...
	utils_test.run_autotest(vm, session, control_path, timeout, outputdir,params, control_args=control_args,ignore_session_terminated=ignore_sess_terminated)
		def run_autotest(vm, session, control_path, timeout,outputdir, params, copy_only=False, control_args=None,ignore_session_terminated=False):
			...
			temp_control_path = config_control(control_path, job_args=job_args)
			vm.copy_files_to(temp_control_path, os.path.join(destination_autotest_path, 'control'))
			...
			session.cmd_output("python -x ./autotest-local %s control & wait ${!}" %verbose,timeout=timeout,print_func=logging.info)
			workspace/lib/python2.7/site-packages/autotest/client/autotest_local.py  ==> I think this file will be called
				...
				from autotest.client import job
					workspace/lib/python2.7/site-packages/autotest/client/job.py
					...
					import glob  ==> here import the glob module..
					...
				def main(self):
					job.runjob(os.path.realpath(self.args[0]), drop_caches, self.options)  ==>Run a job using the given control file.
						myjob = job(control=control, drop_caches=drop_caches, options=options)
						...

for autotest flow, please refer to : http://autotest.readthedocs.io/en/latest/main/local/index.html